clang/docs/ReleaseNotes.rst

```diff
@@ -224,6 +224,9 @@ Bug Fixes in This Version
   for variables created through copy initialization having side-effects in C++17 and later.
   Fixes (#GH64356) (#GH79518).
 
+- Fix value of predefined macro ``__FUNCTION__`` to match MSVC's value. Fixes
+  (`#66114 <https://github.com/llvm/llvm-project/issues/66114>`_).
```

Comment:
```suggestion
- Fix value of predefined macro ``__FUNCTION__`` in MSVC compatibility mode.
  Fixes (#GH66114).
```
I seem to have missed this earlier, but we should probably point out that this change only applies to `MSVCCompat` mode (and also use the new issue link syntax that we have now).